examples/msw/mocks/index.js

```diff
@@ -0,0 +1,15 @@
+const { rest } = require("msw");
+const { setupServer } = require("msw/node");
+
+const handlers = [
+  rest.get("https://my-mock-api.com", (_, res, ctx) => {
+    return res(ctx.status(200), ctx.json({ message: "from msw" }));
+  }),
+];
+
+const server = setupServer(...handlers);
+server.listen({ onUnhandledRequest: "warn" });
+console.info("MSW initialised");
+
+process.once("SIGINT", () => server.close());
+process.once("SIGTERM", () => server.close());
```

Comment:
I don't think this module is used anywhere yet, but we need to take Live Reload into account. We need to ensure that `setupServer` is not called repeatedly, as that will lead to event emitter leaks.

Reply:
I can confirm that it will not be because we're using `-r` to require this file 👍